src/Command.js

```diff
@@ -19,6 +19,47 @@ export default class Command {
     this.concurrency = (!flags || flags.concurrency === undefined) ? DEFAULT_CONCURRENCY : Math.max(1, +flags.concurrency || DEFAULT_CONCURRENCY);
   }
 
+  get name() {
+    // For a class named "FooCommand" this returns "foo".
+    return this.className.replace("Command", "").toLowerCase();
+  }
+
+  get className() {
+    return this.constructor.name;
```

Comment:
Yeah, I see your point.  Feels a little bit magic to base it on the class name.

My thought is that it's easier to get `class *Command` right than it is to set up a name getter.  It also doesn't seem very onerous to require a naming convention for command classes.  They will be relatively few, and they operate in the context of a framework.

How about I add a check for the `Command` suffix on class names and throw an error if a class fails to implement correctly?